.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,19 @@
+name: Snyk
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  security:
+    uses: guardian/.github/.github/workflows/sbt-node-snyk.yml@main
+    defaults:
+      run:
+        working-directory: newswires
```

Comment:
untested, and I can't see how to test before merging, but this looks like it'll run the snyk monitor in the right directory

Reply:
🤞